docs/modules/nexus/pages/administration.adoc

```diff
@@ -0,0 +1,45 @@
+= Nexus Administration
+
+The `ods-core` repository contains a build recipe for a central Nexus instance. The remainder of this document describes how that setup is done, and how it can be modified and updated.
+
+== Updating Nexus to a new version
+
+The Nexus deployment is using an image built in the central ODS namespace. This image is a customisation of the https://hub.docker.com/r/sonatype/nexus3[official Nexus image on DockerHub].
+
+The Nexus version which is used is determined by which image tag from DockerHub is used, and depending on tag, at which time the ODS image is built.
```

Comment:
Perhaps a link to the Dockerfile indicating which tag is being used at the moment? https://github.com/opendevstack/ods-core/blob/master/nexus/docker/Dockerfile

Reply:
Addressed in 2c6cb49. Note that the tag in the `Dockerfile` does not represent what actually gets built. This is determined by the `BuildConfig`.